packages/perseus/src/widgets/categorizer/categorizer.tsx

```diff
@@ -44,7 +49,7 @@ export class Categorizer
     static defaultProps: DefaultProps = {
         items: [],
         categories: [],
-        linterContext: linterContextDefault,
+        linterContext: linterContextDefault as any,
```

Comment:
I couldn't figure this out. I was doing the exact same thing that NumericInput was doing, but I couldn't get rid of the error. Open to suggestions.